mgplvm/crossval/crossval.py

```diff
@@ -72,7 +72,7 @@ def train_cv(mod,
     for p in mod.parameters():  #no gradients for the remaining parameters
         p.requires_grad = False
 
-    if 'EP_GP' in mod.lat_dist.name:
+    if 'GP' in mod.lat_dist.name:
```

Comment:
It would be good to document somewhere the importance of having 'GP' in the model name, maybe in wherever info about defining a new model is / in a readme forthe model class folder. I can otherwise see myself (or someone else) naming somehting without the crucial 'GP' an being very perplexed why it doesn't work.

Reply:
Alternatively, could be a flag that is set as a property of the base class, because that's something that's more searchable when trying to debug.

Reply:
agreed that this is definitely a temporary solution -- I think we're planning to separate out the training and crossvalidation functions for different model classes eventually (issue #59 ) which should resolve this

Reply:
also happy to make this a better temporary solution by doing what you suggest

Reply:
nah this is fine, not worth putting time into changing it at this stage, maybe just adding the comment I suggested down below would be good.